save-core/src/nativeTest/kotlin/org/cqfn/save/core/test.utils/TestUtilsNative.kt

```diff
@@ -0,0 +1,54 @@
+package org.cqfn.save.core.test.utils
+
+import org.cqfn.save.core.files.createFile
+
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.features.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import okio.FileSystem
+
+import kotlinx.coroutines.*
+import kotlinx.coroutines.CoroutineScope
+
+const val timeOut = 100_000L
+
+/**
+ * Workaround to use suspending functions in unit tests
+ *
+ * @param block
+ * @return
+ */
+actual fun runTest(block: suspend (scope: CoroutineScope) -> Unit) = runBlocking { block(this) }
+
+/**
+ * @param url
+ * @param fileName
+ * @return
+ */
+actual suspend fun downloadFile(url: String, fileName: String): String {
```

Comment:
These methods contains quite a lot of common logic, is it possible to refactor it in a short time, in such way, that they will only contain platform dependent code?

Reply:
No chance, different `HttpClient` ktor engines (they have different methods )